ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -74,12 +74,14 @@ class TabBarViewDemoController: DemoController {
         updatedTabBarView.delegate = self
 
         if showsItemTitles {
+            homeItem = TabBarItem(title: "Home", image: UIImage(named: "Home_24")!, selectedImage: UIImage(named: "Home_Selected_24")!)
```

Comment:
Probably better to have a func that creates the items based on a parameter (show titles) and call it here and in the initalization of the homeItem variable?